XLPagerTabStrip/XL/Controllers/XLPagerTabStripViewController.m

```diff
@@ -98,13 +98,6 @@ - (void)viewDidLoad
     }
 }
 
--(void)viewWillAppear:(BOOL)animated
```

Comment:
@mtnbarreto 
Because before this, contentSize is not going to change.
Don't know if I understand correctly